rbs/prism/MethodTypeToParserNodePrism.cc

```diff
@@ -550,17 +545,17 @@ pm_node_t *MethodTypeToParserNodePrism::methodSignature(const pm_node_t *methodD
     auto commentLoc = declaration.commentLoc();
 
     // Pre-calculate frequently used locations to avoid redundant calculations
-    pm_location_t full_loc = PMK::convertLocOffsets(fullTypeLoc);
-    pm_location_t tiny_loc = PMK::convertLocOffsets(firstLineTypeLoc.copyWithZeroLength());
-    pm_location_t end_zero_loc = PMK::convertLocOffsets(fullTypeLoc.copyEndWithZeroLength());
+    pm_location_t full_loc = prism->convertLocOffsets(fullTypeLoc);
+    pm_location_t tiny_loc = prism->convertLocOffsets(firstLineTypeLoc.copyWithZeroLength());
+    pm_location_t end_zero_loc = prism->convertLocOffsets(fullTypeLoc.copyEndWithZeroLength());
 
     // Create receiver: Sorbet::Private::Static
-    pm_node_t *receiver = PMK::SorbetPrivateStatic(fullTypeLoc);
+    pm_node_t *receiver = prism->SorbetPrivateStatic(fullTypeLoc);
     if (!receiver)
         return nullptr;
 
     // Create argument: T::Sig::WithoutRuntime
-    pm_node_t *t_sig_arg = PMK::TSigWithoutRuntime(fullTypeLoc);
+    pm_node_t *t_sig_arg = prism->TSigWithoutRuntime(fullTypeLoc);
```

Comment:
In a separate PR, can you or @KaanOzkan please make our identifiers camelCase?

Sorbet only uses lower_snake_case for external stuff (C++ std, Prism, etc.)